scenarios/kubernetes_e2e.py

```diff
@@ -112,6 +112,24 @@ def kubeadm_version(mode, shared_build_gcs_path):
     else:
         raise ValueError("Unknown kubeadm mode given: %s" % mode)
 
+def kubelet_version(version):
+    """Return gs string to use for kubelet version. Supports ci version aliases ci/latest etc."""
+    if version == 'stable':
+        return 'stable'
+    if version.startswith('ci/'):        
+	version = check_output('gsutil', 'cat', 'gs://kubernetes-release-dev/%s.txt' % version)
+    if not(version.startswith('v')):
+        raise ValueError("Unknown kubelet version given: %s" % version)
+
+    # Work-around for release-1.6 jobs, which still upload debs to an older
+    # location (without os/arch prefixes).
+    # TODO(pipejakob): remove this when we no longer support 1.6.x.
+    if version.startswith('v1.6.'):
+        return 'gs://kubernetes-release-dev/bazel/%s/build/debs/' % version
+
+    # The path given here should match jobs/ci-kubernetes-bazel-build.sh
+    return 'gs://kubernetes-release-dev/bazel/%s/bin/linux/amd64/' % version
```

Comment:
This should be `/ci/%s/...` as we're using `ci/*` version numbers.

Bazel is used for the presubmit job, but that's not working anyway (for other reasons), so we don't have to consider that now

Reply:
Done. Also discovered we do not need the v1.6 condition for ci builds as v1.6 are placed in the same place as the rest.